tests/e2e/core-tests/specs/merchant/wp-admin-product-import-csv.test.js

```diff
@@ -39,15 +40,16 @@ const runImportProductsTest = () => {
 		beforeAll(async () => {
 			await merchant.login();
 			await merchant.openAllProductsView();
-		});
-		it('can upload the CSV file and import products', async () => {
 			await merchant.openImportProducts();
-
-			// Verify error message if you go withot provided CSV file
+		});
+		it('should show error message if you go without providing CSV file', async () => {
+			// Verify error message if you go without providing CSV file
 			await expect(page).toClick('button[value="Continue"]');
 			await page.waitForSelector('div.error');
 			await expect(page).toMatchElement('div.error > p', errorMessage);
```

Comment:
I'm getting consistently `Async callback was not invoked within the 30000ms timeout` errors after this step. I think this is because the page is very briefly blocked after the error notice is displayed.

After I tried temporarily adding `await page.waitFor(2000);` after matching the on the error message the tests passes every time.

Reply:
Thanks for reviewing. I added `Promise.All` for clicking the Continue button.

Reply:
Thanks, works great locally now!